RxSwift/Observables/Implementations/SingleAsync.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  SingleAsync.swift
+//  Rx
+//
+//  Created by Junior B. on 09/11/15.
+//  Copyright © 2015 Krunoslav Zaher. All rights reserved.
+//
+
+import Foundation
+
+class SingleAsyncSink<ElementType, O: ObserverType where O.E == ElementType> : Sink<O>, ObserverType {
+    typealias Parent = SingleAsync<ElementType>
+    typealias E = ElementType
+    
+    private let _parent: Parent
+    private var _seenValue: Bool = false
+    
+    init(parent: Parent, observer: O) {
+        _parent = parent
+        super.init(observer: observer)
+    }
+    
+    func on(event: Event<E>) {
+        switch event {
+        case .Next(let value):
+            
+            if let predicate = _parent._predicate {
+                do {
+                    print("val: \(value)")
```

Comment:
There is a `print` statement accidentally left, I'll push a change to fix this ...